.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,226 @@
+name: acceptance
+on: pull_request
+
+jobs:
+    percynonce:
+      name: Generate Percy nonce
+      runs-on: ubuntu-latest
+      steps:
+        - name: Generate Percy nonce
+          if: always()
+          run: |
+            echo ${{ github.run_id }}.$(($(date +%s))) > nonce.txt
+
+        - name: Upload nonce as artifact
+          if: always()
+          uses: actions/upload-artifact@v1
+          with:
+            name: percy
+            path: nonce.txt
+
+    acceptance:
```

Comment:
@billyvg May I suggest a specific name here? 👀

Reply:
Any suggestions?

Reply:
Ah I don't have strong opinions. 

I suggested this because I was looking at the checkand it didn't seem to provide any context. Im unsure if I'm looking at the right indentation level. 

<img width="664" alt="Screen Shot 2020-05-04 at 5 28 11 PM" src="https://user-images.githubusercontent.com/139499/81015403-a4a1e500-8e2c-11ea-8b7d-e5d7725991f8.png">

Reply:
Ahh, it's because it's only the acceptance tests (running in parallel) --